pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -689,6 +720,67 @@ int socket__http_filter(struct __sk_buff* skb) {
     return 0;
 }
 
+SEC("kprobe/ip_route_output_flow")
+int kprobe__ip_route_output_flow(struct pt_regs* ctx) {
+    struct net *net = (struct net*) PT_REGS_PARM1(ctx);
+    struct flowi4* fl4 = (struct flowi4*) PT_REGS_PARM2(ctx);
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+
+    ip_route_flow_t flow = {};
+    flow.fl = fl4;
+    flow.netns = get_netns(net);
```

Comment:
I think this needs to be `get_netns(&net)`.

Reply:
Thanks, fixed